kitsune/questions/models.py

```diff
@@ -209,7 +209,7 @@ def save(self, update=False, *args, **kwargs):
                     object_id=self.id,
                     creator=self.creator,
```

Comment:
Nit. It seems odd in this case, that the creator of the flagged object is the creator of the question. Maybe we should consider creating a new bot user that we use as the creator of these auto-flagged objects?